src/main/app/src/assets/i18n/nl.json

```diff
@@ -750,6 +750,7 @@
   "msg.error.ztc.client.exception": "Er heeft zich helaas een technische fout voor gedaan. Er kunnen geen zaaktypes worden opgehaald.",
   "msg.error.smartdocuments.not.configured": "SmartDocuments is niet geconfigureerd",
   "msg.error.smartdocuments.disabled": "SmartDocuments is uitgeschakeld",
+  "msg.error.besluit.type": "Verkeerde besluittype",
```

Comment:
```suggestion
  "msg.error.besluit.type": "Verkeerd besluittype",
```

Reply:
On it's own it is without the `e`. If there is a sentence before this, like "this is the wrong ...", then it is correct except for the capital `V`...   So from that I deduce it is on it's own and then my suggested change should be applied 😄